src/Contact/View/page.twig

```diff
@@ -1,5 +1,30 @@
 {% include 'Common/View/header.twig' %}
-<section class="container">
-        <h1>Kontakt</h1>
+<section class="container contact-section">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+        <form method="post" action="" class="form">   
+                <div>
+                <h1 class="form-title">NAPISZ DO NAS</h1> <br>
```

Comment:
You do not need br tag here. h1 is block element and next element will be below it